indico/htdocs/js/indico/modules/categories/management.js

```diff
@@ -301,4 +267,111 @@
             }
         }).isEverythingSelected;
     };
+
+    function _fetchSourceCategory(categoryId) {
+        if (!_categories[categoryId]) {
+            _categories[categoryId] = categoryId;
```

Comment:
I just find it pretty ugly to add unrelated data in the dict to mark it as loading. I didn't look closer but does that code getting stuff from here actually expect an integer in `_categories`? Also, the root category has id 0!